.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
Do you need to add the .prefs files? Or are they specific to each person's setup?

Reply:
I think some of these files are auto generated so we will need to add them to the gitignore file